en/docs/install-and-setup/setup/deployment-best-practices/performance-and-capacity-planning.md

```diff
@@ -0,0 +1,260 @@
+# WSO2 API-M Performance and Capacity Planning
+
+The following sections analyze the results of WSO2 API Manager performance tests.
+
+## Summary
+
+During each release, WSO2 executes various automated performance test scenarios and publishes the results.
+
+| **Test Scenarios** | **Description**                                                      |
+|------------------|--------------------------------------------------------------------------|
+| Passthrough      |   A secured API, which directly invokes the back\-end service\.          |
+| Transformation   |   A secured API, which has a mediation extension to modify the message\. |
+ 
+
+WSO2 uses [Apache JMeter](https://jmeter.apache.org/index.html) as the test client. WSO2 tests each scenario for a fixed duration of time. Thereafter, WSO2 splits the test results into warmup and measurement parts and uses the measurement part to compute the performance metrics.
+
+Test scenarios use a [Netty](https://netty.io/) based back-end service which echoes back any request posted to it after a specified period of time.
+
+WSO2 runs the performance tests with different concurrent user loads, message sizes (payloads), and back-end service delays.
+
+The main performance metrics:
+
+- **Throughput**: The number of requests that the WSO2 API Manager processes during a specific time interval (e.g. per second).
+- **Response Time**: The end-to-end latency for an operation of invoking an API. The complete distribution of response times was recorded.
+
+In addition to the above metrics, WSO2 measures the load average and several memory-related metrics.
+
+The duration of each test is 900 seconds. The warm-up period is 300 seconds. The measurement results are collected after the warm-up period.
+
+A [c5.large Amazon EC2 instance](https://aws.amazon.com/ec2/instance-types/) was used to install WSO2 API Manager.
```

Comment:
Do we need any information from [1] like the "Performance test scripts" section, some info from the "Deployment used for the test" section?

[1] https://github.com/wso2/docs-apim/blob/3.1.0/en/docs/install-and-setup/setup/deployment-best-practices/performance-test-results.md